qfieldsync/gui/cloud_projects_dialog.py

```diff
@@ -100,6 +100,8 @@ def __init__(self, network_manager: CloudNetworkAccessManager, parent: QWidget =
         else:
             self.welcomeLabelValue.setText(self.network_manager.username)
 
+        self.welcomeLabelValue.setToolTip(self.tr(f'You are logged in with the following username at {self.network_manager.url}'))
```

Comment:
Can we do this conditional (only show it when not default)?

Reply:
Done.